spaceship/models/team.py

```diff
@@ -20,6 +20,22 @@ class Team(db.Model):
   created_at = db.Column(PendulumDateTimeField(), default=lambda: pendulum.now('UTC'))
   deleted_at = db.Column(PendulumDateTimeField(), nullable=True)
 
+  def generic_invitation_from(self, inviter):
+    try:
+      generic = next(
+        i for i in self.invitations if (
+          i.inviter == inviter and i.invited_email is None and not i.already_accepted
```

Comment:
why not already_accepted?

Reply:
not sure. we should probably kill invite statuses altogether